src/Symfony/Component/ErrorHandler/ErrorRenderer/HtmlErrorRenderer.php

```diff
@@ -76,7 +79,14 @@ public function render(\Throwable $exception): FlattenException
             $headers['X-Debug-Exception-File'] = rawurlencode($exception->getFile()).':'.$exception->getLine();
         }
 
-        $exception = FlattenException::createFromThrowable($exception, null, $headers);
+        $statusCode = 500;
+        foreach ($this->exceptionsMapping as $exc => $config) {
```

Comment:
Last comment: we could use a more descriptive var name like `$class`.

Reply:
done, thanks!